### PR TITLE
Update pause-on-lock

### PR DESCRIPTION
**Pull request by adrianvg**
Added support for Mate DE.
Tested with Ubuntu 18.04.2 LTS and Mate DE 1.20.1.

### CODE REVIEW

**Review by adrianvg — COMMENTED** [review]
@folixg Did the PR you asked for. Please have a look at it.
Thanks in advance and also for providing the original script!

**Comment by adrianvg** [comment]
Wait a sec, the diff when merged with your code, doesn't look proper.
It jumbled the lines for some reason and put the Mate code in the middle of the X-cinnamon.
Check my fork for clarity. Thanks.

**Comment by folixg** [comment]
Looks good so far. It would be perfect, if you also update the README.

**Comment by adrianvg** [comment]
Will look into the readme during the day.
Thanks!

**Comment by adrianvg** [comment]
Added text and link for Mate in the readme.

**Review by adrianvg — COMMENTED** [review]
@folixg Have a look please. Thanks.

File: pause-on-lock [hunk 1]
```diff
@@ -33,6 +33,10 @@ elif [ "$XDG_CURRENT_DESKTOP" == "X-Cinnamon" ] ; then
   DBUS_LOCK="interface='org.cinnamon.ScreenSaver',member='ActiveChanged'"
   LOCK_SIGNAL="true"
   UNLOCK_SIGNAL="false"
+elif [ "$XDG_CURRENT_DESKTOP" == "MATE" ] ; then
+  DBUS_LOCK="interface='org.mate.ScreenSaver',member='ActiveChanged'"
+  LOCK_SIGNAL="true"
+  UNLOCK_SIGNAL="false"
 else
 echo "Error: Unsupported desktop. Supported desktops are Unity, GNOME and Cinnamon."
```
Comment (folixg) [inline pause-on-lock]: One last line to change, after that everything should be fine.
```suggestion
echo "Error: Unsupported desktop. Supported desktops are Unity, GNOME, Cinnamon and MATE."
```

Comment (adrianvg) [reply pause-on-lock]: Done!

**Comment by folixg** [comment]
Thanks @adrianvg!

**Comment by adrianvg** [comment]
> Thanks @adrianvg!

No prob, glad to help. Self-interest you know. ;-)

**Pull request merged**